interface/billing/billing_report.php

```diff
@@ -659,6 +669,13 @@ function criteriaSelectHasValue(select) {
                         );
                         $TPSCriteriaKeyMaster = "form_encounter.date,billing.date,claims.process_time,claims.target,patient_data.fname," . "form_encounter.pid,claims.payer_id,form_encounter.encounter,insurance_data.provider,billing.id,billing.billed," . "billing.authorized,form_encounter.last_level_billed,billing.x12_partner_id";
                         $TPSCriteriaDataTypeMaster = "datetime,datetime,datetime,radio,text_like," . "text,include,text,radio,radio,radio," . "radio_like,radio,query_drop_down";
+                        $combined = array_map(null, $TPSCriteriaDisplayMaster, explode(",", $TPSCriteriaKeyMaster), explode(",", $TPSCriteriaDataTypeMaster));
+usort($combined, function($a, $b) {
+    return strcmp($a[0], $b[0]);
+});
+$TPSCriteriaDisplayMaster = array_column($combined, 0);
+$TPSCriteriaKeyMaster = implode(",", array_column($combined, 1));
+$TPSCriteriaDataTypeMaster = implode(",", array_column($combined, 2));
```

Comment:
hi @sjdevgo, thanks for the PR! It will be a valuable contribution.

Unfortunately the code below here has hard coded logic for the radio types so it's going to not associate correctly. Maybe you can figure out a way to get around this since hard coding isn't a very good solution?